### PR TITLE
added first things

### PR DESCRIPTION
**Pull request by iwonaive**

### CODE REVIEW

File: server.ts [hunk 1]
```diff
@@ -0,0 +1,56 @@
+import { APP_BASE_HREF } from '@angular/common';
+import { CommonEngine } from '@angular/ssr';
+import express from 'express';
+import { fileURLToPath } from 'node:url';
+import { dirname, join, resolve } from 'node:path';
+import bootstrap from './src/main.server';
+
+// The Express app is exported so that it can be used by serverless Functions.
+export function app(): express.Express {
```
Comment (devolajf) [inline server.ts:9]: why did you decide to add a custom server?

File: src/app/app.config.ts [hunk 2]
```diff
@@ -0,0 +1,10 @@
+import { ApplicationConfig } from '@angular/core';
+import { provideRouter } from '@angular/router';
+import { provideHttpClient } from '@angular/common/http';
+
+import { routes } from './app.routes';
+import { provideClientHydration } from '@angular/platform-browser';
+
+export const appConfig: ApplicationConfig = {
+  providers: [provideRouter(routes), provideClientHydration(), provideHttpClient()]
```
Comment (devolajf) [inline src/app/app.config.ts:9]: I'm not sure if you're need provideClientHydration :)

File: src/app/components/footer/footer.component.ts [hunk 3]
```diff
@@ -0,0 +1,10 @@
+import { Component } from "@angular/core";
+
+@Component({
+    standalone: true,
```
Comment (devolajf) [inline src/app/components/footer/footer.component.ts:4]: you can use ng lint to check indentation for you

File: src/app/interfaces/charminglook.ts [hunk 4]
```diff
@@ -0,0 +1,16 @@
+export interface Charminglook {
```
Comment (devolajf) [inline src/app/interfaces/charminglook.ts:1]: great 👍

File: src/app/services/data.service.ts [hunk 5]
```diff
@@ -0,0 +1,7 @@
+import { Injectable } from '@angular/core';
+import { HttpClient } from '@angular/common/http';
+
+@Injectable({ providedIn: 'root' })
+export class DataService {
```
Comment (devolajf) [inline src/app/services/data.service.ts:5]: well done

**Review by devolajf — APPROVED** [review]
Great job 👏

File: server.ts (hunk 1 above)
Comment (iwonaive) [reply server.ts:9]: I did not ;) I mean it was there, I did't touch it

File: src/app/app.config.ts (hunk 2 above)
Comment (iwonaive) [reply src/app/app.config.ts:9]: This was also there already, so I didn't delete it. You think that Angular is playing with me? ;)

File: src/app/components/footer/footer.component.ts (hunk 3 above)
Comment (iwonaive) [reply src/app/components/footer/footer.component.ts:4]: Ok, I'll learn about it

**Pull request merged**